Gopkg.lock

```diff
@@ -2,14 +2,175 @@
 
 
 [[projects]]
+  branch = "master"
+  name = "github.com/Azure/go-ansiterm"
+  packages = [".","winterm"]
+  revision = "fa152c58bc15761d0200cb75fe958b89a9d4888e"
+
+[[projects]]
+  name = "github.com/Microsoft/go-winio"
```

Comment:
Seeing these makes me wonder if we need to have something split into a separate interface with build-tag include implementations.  What are these dependencies being used for?